### PR TITLE
Fix for npmjs.com

### PR DESCRIPTION
**Pull request by Yash-Singh1**
- Inverts `git` repository icon for packages

### CODE REVIEW

**Comment by Gusted** [comment]
Hey @Yash-Singh1 

Do you have an example URL?

Regards,
Gusted

**Comment by Yash-Singh1** [comment]
https://www.npmjs.com/package/lodash

**Comment by Gusted** [comment]
image
Seems for me that it's already being styled correctly.

**Comment by Yash-Singh1** [comment]
@Gusted Try it in Chrome. I tried it out and sometimes after I reload the `git` icon seems to be inverted correctly while at other times it is just black.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5509,6 +5509,7 @@ header a[href="/"] svg
 #customers_panel img[src*="visa.full.png"]
 ._5532dff2
 ._93bbf0b4
+#top > div > div:nth-child(9) > p > a > span > svg
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5512]:
```CSS
a title + g[data-darkreader-inline-fill] {
    fill: var(--darkreader-neutral-text) !important;
}
```
This should work for both Firefox and Chrome and have the correct color.

**Pull request merged**